lab-davide/app/entry.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
```

Comment:
To me, an entry file is backend code that provides instructions for webpack. It shouldn't contain any frontend code, like the stuff you're doing below to define the module--that should be handled in a separate, appropriately named file which you would then require in here.